### PR TITLE
518 codelist same as bug

### PR DESCRIPTION
**Pull request by GDonRanasinghe**
Fixed the output csv not having the same as uri column when the concepts defined elsewhere are used.

### CODE REVIEW

File: tests/unit/models/test_code_list_config.py [hunk 1]
```diff
@@ -166,3 +182,68 @@ def test_code_list_config_with_concepts_defined_elsewhere():
...
+        output_dir = temp_dir / "out"
+        config_path = _test_case_writers_base_dir / "skoscodelistwriter" / "config.json"
+        csv_path = _test_case_writers_base_dir / "skoscodelistwriter" / "data.csv"
+
+        cli_build(
+            output_directory=output_dir, csv_path=csv_path, config_path=config_path
+        )
+
+        output_df = pd.read_csv(output_dir / "geography.csv")
+        assert "Original Concept URI" in output_df.columns
```
Comment (robons) [inline tests/unit/models/test_code_list_config.py:249]: Can you make sure the URIs you're expecting get serialised?

**Review by robons — APPROVED** [review]
Approved on the condition that you test that the same as URIs defined in the JSON file are definitely in the data frame/CSV.

**Review by canwaf — CHANGES_REQUESTED** [review]
This change has regressed the functionality for the `from_template: year` configuration option.

Additionally, you have put live data in your tests. Please synthesise more narrow test cases for unit tests.

I need to have the from_template functionality working. Let's talk about how the serialisation has changed and how we can improve our test coverage to prevent future regression from getting to this point.

**Review by canwaf — APPROVED** [review]
Thank you @CharlesRendle for making generic tests for this; and only testing one instance (since we only need to test things once).

**Pull request merged**